src/fontra/views/editor/scene-controller.js

```diff
@@ -647,18 +647,37 @@ export class SceneController {
     this.contextMenuState.openContourSelection = glyphController.canEdit
       ? getSelectedClosableContours(glyphController.instance.path, pointSelection)
       : [];
+    this.contextMenuState.joinContourSelection = glyphController.canEdit
+      ? getSelectedJoinContoursPointIndices(
+          glyphController.instance.path,
+          pointSelection
+        )
+      : [];
   }
 
   getContextMenuItems(event) {
     const contextMenuItems = [
       {
-        title: () =>
-          translatePlural(
-            "action.close-contour",
-            this.contextMenuState.openContourSelection?.length
-          ),
-        enabled: () => this.contextMenuState.openContourSelection?.length,
-        callback: () => this.doCloseSelectedOpenContours(),
+        title: () => {
+          if (this.contextMenuState.joinContourSelection?.length === 2) {
```

Comment:
This also triggers when the two selected points are part of the same contour. Which leads to the wrong title.

Also, this should be a `condition ? A : B` expression.

Reply:
Makes absolutely sense, with the 'same contour' issue. The was even duplicated/redundant code related to that, I was able to remove. Sorry. The condition is done, too.